src/jsMain/resources/index.html

```diff
@@ -4,52 +4,14 @@
   <meta charset="UTF-8">
   <title>Simulator | sparklemotion</title>
   <link rel="stylesheet" href="styles.css"/>
+  <link rel="stylesheet" href="react-mosaic-component.css"/>
   <link rel="stylesheet" href="./fontawesome-free/css/all.min.css">
+  <link href="https://unpkg.com/normalize.css@^7.0.0" rel="stylesheet" />
```

Comment:
Super minor thing, but should we move these off a CDN in the event we forget to do it before playa?

Reply:
Ah, that's a great point! We should definitely host it locally